common/index.ts

```diff
@@ -0,0 +1,40 @@
+import AdmZip from 'adm-zip';
+import fs from 'fs';
+import fetch from 'isomorphic-fetch';
+
+export enum ProjectSource {
```

Comment:
This enum is kind of weird. The first three values are option fields. The next value is a filename found in sb2 and sb3 zips. The last is a url prefix.

This enum should probably be three.